akka-projection-core/src/main/scala/akka/projection/internal/OffsetSerialization.scala

```diff
@@ -8,43 +8,75 @@ import java.util.UUID
 
 import akka.annotation.InternalApi
 import akka.persistence.query
+import akka.projection.ProjectionId
 
 /**
  * INTERNAL API
  */
 @InternalApi private[akka] object OffsetSerialization {
+  sealed trait StorageRepresentation
+  final case class SingleOffset(id: ProjectionId, manifest: String, offsetStr: String, mergeable: Boolean = false)
+      extends StorageRepresentation
+  final case class MultipleOffsets(reps: Seq[SingleOffset]) extends StorageRepresentation
```

Comment:
use `import scala.collection.immutable`, and `immutable.Seq` here, since `Seq` from predef is possibly mutable

Reply:
:+1: